ui/window.go

```diff
@@ -2117,6 +2141,35 @@ func (window *Window) handleGlobalShortcuts(event *tcell.EventKey) *tcell.EventK
 		return nil
 	}
 
+	if window.app.VimMode.CurrentMode != vim.Disabled {
```

Comment:
What about the login view, I assume that's not handled with this?

Reply:
Login is not handled.